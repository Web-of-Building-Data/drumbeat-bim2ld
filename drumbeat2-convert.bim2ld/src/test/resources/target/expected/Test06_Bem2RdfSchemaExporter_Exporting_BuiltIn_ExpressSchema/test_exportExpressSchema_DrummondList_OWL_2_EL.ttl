@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix EXPRESS: <http://drumbeat.cs.hut.fi/owl/EXPRESS#> .
@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .

EXPRESS:Select  a  owl:Class .

EXPRESS:BOOLEAN  a  owl:Class .

EXPRESS:Collection  a  owl:Class .

EXPRESS:hasBinary  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:INTEGER  a  owl:Class .

EXPRESS:EmptyList  a     owl:Class ;
        rdfs:subClassOf  EXPRESS:List .

EXPRESS:LOGICAL  a  owl:Class .

EXPRESS:Enum  a  owl:Class .

EXPRESS:Entity  a  owl:Class .

EXPRESS:STRING  a  owl:Class .

EXPRESS:hasNext  a   owl:ObjectProperty ;
        rdfs:domain  EXPRESS:Collection ;
        rdfs:range   EXPRESS:Collection .

EXPRESS:EmptyBag  a      owl:Class ;
        rdfs:subClassOf  EXPRESS:Bag .

EXPRESS:NUMBER  a  owl:Class .

EXPRESS:hasValue  a  owl:DatatypeProperty .

EXPRESS:EmptySet  a      owl:Class ;
        rdfs:subClassOf  EXPRESS:Set .

EXPRESS:BINARY  a  owl:Class .

EXPRESS:SpecialValue  a  owl:Class .

EXPRESS:hasContent  a  owl:ObjectProperty ;
        rdfs:domain  EXPRESS:Collection .

EXPRESS:REAL  a  owl:Class .

EXPRESS:Defined  a  owl:Class .

EXPRESS:UNKNOWN  a  EXPRESS:LOGICAL .

EXPRESS:Bag  a           owl:Class ;
        rdfs:subClassOf  EXPRESS:Collection .

EXPRESS:hasInteger  a       owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:Set  a           owl:Class ;
        rdfs:subClassOf  EXPRESS:Collection .

EXPRESS:hasReal  a          owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:DATETIME  a  owl:Class .

EXPRESS:Array  a         owl:Class ;
        rdfs:subClassOf  EXPRESS:Collection .

EXPRESS:List  a          owl:Class ;
        rdfs:subClassOf  EXPRESS:Collection .

EXPRESS:hasDatetime  a      owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:FALSE  a  EXPRESS:BOOLEAN , EXPRESS:LOGICAL .

EXPRESS:TRUE  a  EXPRESS:BOOLEAN , EXPRESS:LOGICAL .

EXPRESS:hasString  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:hasNumber  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:EmptyArray  a    owl:Class ;
        rdfs:subClassOf  EXPRESS:Array .
