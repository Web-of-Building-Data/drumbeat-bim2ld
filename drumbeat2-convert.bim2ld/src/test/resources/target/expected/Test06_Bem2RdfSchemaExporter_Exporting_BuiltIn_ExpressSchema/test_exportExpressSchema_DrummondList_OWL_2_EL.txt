@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix EXPRESS: <http://drumbeat.cs.hut.fi/owl/EXPRESS#> .
@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .

EXPRESS:Select  a  owl:Class .

EXPRESS:BOOLEAN  a  owl:Class .

EXPRESS:hasBinary  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:INTEGER  a  owl:Class .

EXPRESS:EmptyList  a     owl:Class ;
        rdfs:subClassOf  EXPRESS:List .

EXPRESS:LOGICAL  a  owl:Class .

EXPRESS:Enum  a  owl:Class .

EXPRESS:Entity  a  owl:Class .

EXPRESS:hasNext  a   owl:ObjectProperty ;
        rdfs:domain  EXPRESS:List ;
        rdfs:range   EXPRESS:List .

EXPRESS:STRING  a  owl:Class .

EXPRESS:NUMBER  a  owl:Class .

EXPRESS:hasValue  a  owl:DatatypeProperty .

EXPRESS:BINARY  a  owl:Class .

EXPRESS:hasContent  a  owl:ObjectProperty ;
        rdfs:domain  EXPRESS:List .

EXPRESS:REAL  a  owl:Class .

EXPRESS:Defined  a  owl:Class .

EXPRESS:UNKNOWN  a  EXPRESS:LOGICAL .

EXPRESS:hasInteger  a       owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:hasReal  a          owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:DATETIME  a  owl:Class .

EXPRESS:List  a  owl:Class .

EXPRESS:hasDatetime  a      owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:FALSE  a  EXPRESS:BOOLEAN , EXPRESS:LOGICAL .

EXPRESS:TRUE  a  EXPRESS:BOOLEAN , EXPRESS:LOGICAL .

EXPRESS:hasString  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .

EXPRESS:hasNumber  a        owl:DatatypeProperty ;
        rdfs:subPropertyOf  EXPRESS:hasValue .
